wts/blueprints/oauth2.py

```diff
@@ -49,11 +51,17 @@ def get_authorization_url():
 
     requested_idp = flask.request.args.get("idp", "default")
     client = get_oauth_client(idp=requested_idp)
-    # This will be the value that was put in the ``client_kwargs`` in config.
-    redirect_uri = client.client_kwargs.get("redirect_uri")
+    # This will be the value that was put in the ``metadata`` in config.
+    state_prefix = client.metadata.get("state_prefix")
+    authorize_url = client.metadata.get("authorize_url")
+    state = generate_token()
+    if state_prefix:
+        state = state_prefix + "-" + state
```

Comment:
what is `state_prefix` for?

Reply:
nvm i understand after reading https://github.com/uc-cdis/cloud-automation/pull/1391

Reply:
would be cool to document it though, in the code or readme

Reply:
the `cloud-automation` PR keys on state_prefix to know which child-account to redirect to